### PR TITLE
Anolis instructions

### PR DESCRIPTION
**Pull request by nwtn**

### CODE REVIEW

**Comment by nwtn** [comment]
just updated this a wee bit with some info about html tidy

**Comment by marcoscaceres** [comment]
@nwtn good call about tidy! Just saw someone else had issues on IRC.

File: README.md [hunk 1]
```diff
@@ -1,3 +1,76 @@
...
+    (Note that Python will probably show a bunch of warnings, but as long as
+    there are no *errors*, you should be OK.)
+
+That's it for installing Anolis. To generate the page, just go to your picture
+repo and run `./gen.js`.
+
+If you get something like "Error: Cannot find module
+'optimist'", try doing the `sudo npm install -g optimist` in your home directory.
+
+If you get something like "Missing Dependency, yay! Please install HTML5 Tidy: https://github.com/w3c/tidy-html5" (even though you've installed it), try running `tidy -version`; if the version listed isn't the HTML5 version, then `tidy` is pointing to a different/old version, not the one you just installed. You may need to update your path, or you may be able to replace the old one in `/usr/bin/` with the new one in `/usr/local/bin/`.
```
Comment (marcoscaceres) [inline README.md]: Please 80 col this :) Then we are good to merge this badboy!

**Comment by nwtn** [comment]
sorry about that. should be good to go now.

**Pull request merged**